arthimetic-operators.js

```diff
@@ -28,3 +34,19 @@ Task 1:
    5. Divide two numbers and round the result to the nearest integer before logging it to the console.
 *******************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+
+let x= 4-3
+console.log(x)
+
+let y= 1*3*5*7
+console.log(y);
+
+let rectangular= 3*5*7
+console.log(rectangular); 
+let declare=9.99%0.20
+
+console.log(declare);
+
+let s= 10/3
```

Comment:
This is not what was required, please declare two variables like I mentioned in the comments above.
Then create a new variable called `result` which would divide the two numbers and round the result to the nearest integer using `Math.round()`